### PR TITLE
Verify us citizenship across all household members

### PR DESCRIPTION
**Pull request by jayroh**
If the primary client answers that there is someone in the household that
is not a US citizen then they should be taken to a page that asks them
*who* is not a citizen.

### CODE REVIEW

File: app/views/medicaid/intro_citizen/edit.html.erb [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 <div class="form-card">
   <header class="form-card__header">
-    <div class="form-card__title">Are you currently a US Citizen?</div>
+    <div class="form-card__title">
+      Is everyone in your household currently a US citizen?
```
Comment (jessieay) [inline app/views/medicaid/intro_citizen/edit.html.erb]: do we want to show the other page title if there is just one person in the household? We did that using i18n pluralization for other features

File: app/controllers/medicaid/citizens_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    def skip?
+      everyone_a_citizen?
+    end
+
+    def member_attrs
+      %i[citizen]
+    end
+
+    def everyone_a_citizen?
+      current_application.citizen?
```
Comment (jessieay) [inline app/controllers/medicaid/citizens_controller.rb]: while are are here, does it make sense to rename the `citizen` column to `everyone_a_citizen` on medicaid apps? that's the name it has on on snap apps and we've been trying to standardize as we go when possible

File: spec/features/medicaid_application_with_multiple_members_spec.rb [hunk 3]
```diff
@@ -48,7 +48,9 @@
     end
 
     on_page "Introduction" do
-      expect(page).to have_content("Are you currently a US Citizen?")
+      expect(page).to have_content(
+        "Is everyone in your household currently a US citizen",
+      )
       click_on "Yes"
```
Comment (jessieay) [inline spec/features/medicaid_application_with_multiple_members_spec.rb]: Maybe we should say `no` here so we have a chance to indicate who is *not* a citizen in a feature spec?

File: spec/controllers/medicaid/citizens_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+          )
+        session[:medicaid_application_id] = medicaid_application.id
+
+        get :edit
+
+        expect(response).to redirect_to(subject.next_path)
+      end
+    end
+
+    context "someone in the home is not a use citizen" do
```
Comment (jessieay) [inline spec/controllers/medicaid/citizens_controller_spec.rb]: typo `use` -> `US`

File: spec/controllers/medicaid/intro_citizen_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe Medicaid::IntroCitizenController, type: :controller do
+  describe "#next_path" do
```
Comment (jessieay) [inline spec/controllers/medicaid/intro_citizen_controller_spec.rb]: may want to look at the specs for `IntroCollegeController` -- we make sure that the member for a single member household is updated via this controller, which I think we should be doing for all single member households on these "yes/no" pages.

**Comment by jayroh** [comment]
[[Medicaid MM Flow] Intro: Is everyone in your household currently a US Citizen? + Nested](https://trello.com/c/zouJo1sW/432-medicaid-mm-flow-intro-is-everyone-in-your-household-currently-a-us-citizen-nested)

**Pull request merged**